users/static/users/scripts/float_menu.js

```diff
@@ -1,9 +1,42 @@
-$(document).ready(function () {
+var previous_position
+
+function range(size, startAt = 0) {
+    return [...Array(size).keys()].map(i => i + startAt);
+}
+
+function borderCrossed(xPos, border) {
+    var size = Math.abs(previous_position - xPos) + 1
+    var array = range(size, Math.min(previous_position, xPos))
+    var is_border_in_range_bounds = array[0] == border
+    console.log(array)
+    console.log(border)
+    console.log($(window).width())
+    if (size > 1) {
+        return array.includes(border) && !is_border_in_range_bounds
+    }
+    else {
+        return array.includes(border) && previous_position != border
+    }
+}
+
+$(document).ready(function() {
+    previous_position = $(window).width()
```

Comment:
Convention in javascript says that variables should be names with CamelCase instead of snake_case :) Please change it

Reply:
Done